### PR TITLE
[Feature] add enable native flag when Initializing webRTC

### LINKED ISSUE #471: [REQUEST] How to get logs genereated by libwebrtc
**Is your feature request related to a problem? Please describe.**
Hi, @karasusan 

I want to ask how to get logs genereated by libwebrtc. 
For I didn't find any implementation of `rtc::Log` related, I guess that logs of libwebrtc is streamed to `stderr` by default. But either in `Editor.log` or `Player.log` I can't find any contents of `stderr`.

Currently I implemented `rtc::LogSink` interface to stream logs to a hard-coded file and make it inuse by `rtc::LogMessage::AddLogToStream` . For logs of libwebrtc is quite useful while troubleshooting, in my view a more completed solution is needed here.

**Describe the solution you'd like**
Expose user interface and register C# callback to `rtc::LogSink`, so log behavior of libwebrtc can be customized.

**Describe alternatives you've considered**
Redirect `stderr` to any existing Unity logs

### PR DESCRIPTION
**Pull request by juju812**
fix #471

### CODE REVIEW

File: Plugin~/WebRTCPlugin/UnityLogStream.cpp [hunk 1]
```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```
Comment (karasusan) [inline Plugin~/WebRTCPlugin/UnityLogStream.cpp:1]: Is this comment needed?
The source code license is under UCL.
https://github.com/Unity-Technologies/com.unity.webrtc/blob/develop/LICENSE.md

Comment (karasusan) [reply Plugin~/WebRTCPlugin/UnityLogStream.cpp:1]: https://github.com/Unity-Technologies/com.unity.webrtc/pull/519/files#diff-1e4a8f07211ad301d32a15a93d1511e033b7d5d56a6586cf3dc393fc9a0fce7bR1

Comment (juju812) [reply Plugin~/WebRTCPlugin/UnityLogStream.cpp]: removed license header generated by VS

**Comment by karasusan** [comment]
I confirmed this pull request and found the compile error.

I fixed the error and pushed to the origin repository.
Could you put the commit to this pull request?
https://github.com/Unity-Technologies/com.unity.webrtc/commit/1e9591d821d9e4e4df5b0181e60417469c088b6f

**Comment by juju812** [comment]
@karasusan

For our working branch is detached with mainline of this repo, I didn't noticed compile error while syncing the changes. Sorry for my careless and it's fixed now.

We're planning to develop features such as Nvidia HW H264 decoding and online changing resolution. After that we'll merge all changes to the mainline.

**Review by karasusan — APPROVED** [review]

**Pull request merged**